surch/logger.py

```diff
@@ -58,9 +58,8 @@ def configure():
         sys.exit('Failed retrieving log file path ({0}).'.format(str(ex)))
```

Comment:
we should consider removing this entirely stupid logger thing and generate a simple logger in `utils` or something. Then we can dump `six.py`, `dictconfig.py` and `logger.py`